Cargo.toml

```diff
@@ -16,6 +16,10 @@ categories = ["database-implementations"]
 crate-type = ["cdylib", "rlib"]
 name = "rejson"
 
+[dev-dependencies]
+env_logger = "0.10.0"
+redis-module = { version="1.0", features = ["experimental-api", "test"]}
```

Comment:
Lets just make sure this will not somehow effect the final .so file itself, we need to see that the redisjson.so are still using the Redis allocator.

Reply:
I'll comment it out